### ISSUE TITLE
USWDS - Bug: H4 headings appearing above H1

### ISSUE THREAD

**Issue by amycole501** [uswds/uswds#5488]
### Describe the bug

On 139 USWDS pages, the H4 heading on the top right titled "on this page" appears above the H1. This violates [WCAG success criteria 2.4.6 headings and labels](https://www.w3.org/TR/UNDERSTANDING-WCAG20/navigation-mechanisms-descriptive.html).

### Steps to reproduce the bug

1. Go to any page with "on this page" in-page navigation. Example: https://designsystem.digital.gov/documentation/migration/
2. Use [ANDI](https://www.ssa.gov/accessibility/andi/help/install.html) to inspect the page structure
3. See resulting page hierarchy:
![image](https://github.com/uswds/uswds-site/assets/117778073/a14d2c5e-e76b-4c7f-b322-abaa60a3be1c)

### Expected Behavior

The H1 should appear first in order on every page. In this case, the "Migrating to USWDS 3.0" should be first on the page. 

### System setup

This occurs in all browsers and operating systems.

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://designsystem.digital.gov/about/community/#community-conduct).
- [X] I checked the [current issues](https://github.com/uswds/uswds-site/issues) for duplicate bug reports.

**Comment by amyleadem** [comment]
Note: this issue is a duplicate of https://github.com/uswds/uswds/issues/5279 but I am keeping this one open since it has a bit more detail.

**Comment by mejiaj** [comment]
@amycole501 @alex-hull can you confirm this is still an issue? We haven't seen this issue flagged with our other testing tools.

**Comment by mahoneycm** [comment]
@amycole501 It looks like W3C actually deems this as acceptable because they are separate fixed sections of the page:

> **Exception for fixed page sections**
In fixed sections of the page, for example in sidebars, the heading ranks should not change depending on the ranks in the content area. In those cases, consistency across pages is more important. [^1]

[^1]: [Heading structure exception for fixed page sections](https://arc.net/l/quote/zdcbazdc)

They also provide a similar [example](https://arc.net/l/quote/guacvtbq) on that page

<img width="713" alt="image" src="https://github.com/uswds/uswds/assets/25211650/cf97dbde-f051-46a1-88b1-801ec6da3d46">

**Comment by amycole501** [comment]
Good find! We can remove this issue. When it appears in our automated checkers, we can dismiss it.

**Comment by mahoneycm** [comment]
Sounds good to me! Closing this issue 👍
